pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -48,4 +49,8 @@ type DeschedulerConfiguration struct {
 
 	// EvictLocalStoragePods allows pods using local storage to be evicted.
 	EvictLocalStoragePods bool `json:"evictLocalStoragePods,omitempty"`
+
+	// Logging specifies the options of logging.
+	// Refer [Logs Options](https://github.com/kubernetes/component-base/blob/master/logs/options.go) for more information.
+	Logging componentbaseconfig.LoggingConfiguration
```

Comment:
It seems `make gen` hasn't been executed last time we change the component config api, so we didn't notice this missing member.

BTW, do we need to use versioned apis for component config? Because I noticed that flags were directly added to component config rather than being added to versioned config.

Reply:
> BTW, do we need to use versioned apis for component config? Because I noticed that flags were directly added to component config rather than being added to versioned config.

since this componentconfig api is currently only used internally by the descheduler, I don't think we technically need to maintain any versioning for it. However it would be good to keep the versioning up to date if we plan to expose this api at any point (as suggested in https://github.com/kubernetes-sigs/descheduler/issues/431)